real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/SensorSwitchPanel.java

```diff
@@ -0,0 +1,40 @@
+package com.roboclub.robobuggy.ui;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.GridLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.BorderFactory;
+import javax.swing.JButton;
+import javax.swing.JPanel;
+
+import com.roboclub.robobuggy.ros.SensorChannel;
+
+public class SensorSwitchPanel extends RoboBuggyGUIContainer{
+	
+	SensorSwitch gps_switch;
+	SensorSwitch vision_switch;
+	SensorSwitch encoders_switch;
+	SensorSwitch imu_switch;
+	SensorSwitch controls_switch;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1747252181/issues/source?bid=2938665&fileBranchId=3059924#l21)